src/sdsc/__init__.py

```diff
@@ -1141,6 +1170,9 @@ def initialize():
             checkmodule = os.path.splitext(os.path.basename(checkfile))[0]
             transform = etree.XSLT(etree.parse(checkfile, parser))
             prepared_checks.append({'name': checkmodule, 'transform': transform})
+        # FIXME toms (style): not good to check for Exception. This can hide too much.
+        # Better check for specific exceptions like XMLSyntaxError, XSLTParseError,
+        # XSLTApplyError, XSLTExtensionError, ...?
```

Comment:
Well... study my code on several of my Python projects. I *never, ever* check for `Exception`, neither do I need to check for "twenty different exception" (to cite you). 😉

I see your point. However, if you structure your code cleverly, only check what you really want to check. Not more, not less. Delegate everything else to some upper layer.

With this strategy I keep my code clean and don't hide any error. In the case that I don't catch an error, the script will fail of course. **Let it fail!** You **want** to see it fail, because you want to know the error. Catching *any* exception hides that. This is not dogma, but about maintenance and traceability of errors.

Sorry, to say that, but this line opens pandora's box. 😉